servers/dsub/jobs/controllers/utils/query_parameters.py

```diff
@@ -17,20 +17,34 @@ def api_to_dsub(query):
 
     dstat_params = {}
 
-    epoch = datetime.datetime.utcfromtimestamp(0).replace(tzinfo=tzutc())
-    dstat_params['create_time'] = int(
-        (query.start - epoch).total_seconds()) if query.start else None
-
-    dstat_params['job_names'] = {query.name} if query.name else None
-
     dstat_params['statuses'] = {
         job_statuses.api_to_dsub(s)
         for s in query.statuses
     } if query.statuses else {'*'}
 
-    dstat_params['labels'] = {
-        param_util.LabelParam(k, v)
-        for (k, v) in query.labels.items()
-    } if query.labels else None
+    if query.start:
+        epoch = datetime.datetime.utcfromtimestamp(0).replace(tzinfo=tzutc())
+        dstat_params['create_time'] = int(
+            (query.start - epoch).total_seconds())
+    if query.name:
+        dstat_params['job_names'] = {query.name}
+    if query.labels:
+        dstat_params['job_ids'] = {
+            v
+            for (k, v) in query.labels.items() if k == 'job-id'
```

Comment:
Isn't query.labels a dict? Why not just index into it by `'job-id'`?

Reply:
Good point, I was thinking there could be multiple job-id's, but that isn't supported by our API right now. Switched to index directly for each reserved label.